tools/3dtrees_detailview/.shed.yml

```
@@ -0,0 +1,9 @@
+name: 3dtrees_detailview
+owner: bgruening
+description: species prediction on segmented point cloud.
+homepage_url: https://github.com/3dTrees-earth/3dtrees-detailview
+long_description: |
+  Performs tree species classification on segmented 3D point clouds using the DetailView model.
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/3dtrees_detailview
+type: unrestricted
+categories: Geo Science
```

```suggestion
categories:
- "Geo Science"
```

Ich glaube das musste ich auch bei den anderen machen